packages/components/package.json

```diff
@@ -70,6 +70,7 @@
     "react-router-dom": "~6.27.0",
     "react-select": "~5.8.2",
     "react-treebeard": "~3.2.4",
+    "status": "^0.0.13",
```

Comment:
was this change in the package.json intentional? Seems unrelated and maybe a typo